docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -81,6 +68,7 @@ we are not provided).
 
 
 ## Lag and Backfill
-Google does not update the search data daily, but has an uncertain update
-frequency. The delay can range from 1 day to 10 days or even more. We check for
+Google does not currently update the search data daily, but usually twice a week.
+Each update will usually extend the coverage to within three days of the day of the update.
+As a result the delay can range from 3 to 10 days or even more. We check for
 updates every day and provide the most up-to-date data.
```

Comment:
This section is listed in the template, but never made it into the page I guess? It would be nice to include a summary of what people are allowed to do with the data (who do they have to cite? can they sell anything they make using the data? do they have to share what they make under the same terms?) but not strictly necessary. 

Essentially we want people to be able to use this page to quickly evaluate whether this dataset is likely meet their needs, so that they can then decide whether it justifies reading a bunch of legal text.

```suggestion
updates every day and provide the most up-to-date data.

## Source and Licensing
This dataset is based on Google's [COVID-19 Search Trends symptoms dataset](http://goo.gle/covid19symptomdataset), which is licensed under Google's [Terms of Service](https://policies.google.com/terms).

To learn more about the source data, how it is generated and its limitations, 
read [Google's Search Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```